spec/presenters/hyrax/collection_presenter_spec.rb

```diff
@@ -120,29 +111,38 @@
   end
 
   describe "#total_items", :clean_repo do
-    subject { presenter.total_items }
-
     context "empty collection" do
-      it { is_expected.to eq 0 }
+      let(:collection) { FactoryBot.valkyrie_create(:hyrax_collection) }
+
+      it 'returns 0' do
+        expect(presenter.total_items).to eq 0
+      end
     end
 
-    context "collection with work" do
-      let!(:work) { create(:work, member_of_collections: [collection]) }
+    context "collection with works" do
+      let(:collection) { FactoryBot.valkyrie_create(:hyrax_collection) }
+      let!(:work) { FactoryBot.valkyrie_create(:hyrax_work, member_of_collection_ids: [collection.id]) }
 
-      it { is_expected.to eq 1 }
+      it 'returns 1' do
+        expect(presenter.total_items).to eq 1
+      end
     end
 
     context "null members" do
       let(:presenter) { described_class.new(SolrDocument.new(id: '123'), nil) }
 
-      it { is_expected.to eq 0 }
+      it 'returns 0' do
+        expect(presenter.total_items).to eq 0
+      end
     end
   end
 
   describe "#total_viewable_items", :clean_repo do
     subject { presenter.total_viewable_items }
     let(:ability) { double(::Ability, user_groups: ['public'], current_user: user) }
     let(:user) { create(:user) }
+    let(:collection) { FactoryBot.create(:collection_lw) }
```

Comment:
Is it intentional for this to be `collection_lw`?  I would have expected this to be `hyrax_collection` based on other changes in this spec.  Should it be...

```suggestion
    let(:collection) { FactoryBot.create(:hyrax_collection) }
```